kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_with_constraints.h

```diff
@@ -1444,12 +1430,10 @@ class ResidualBasedEliminationBuilderAndSolverWithConstraints
         mReactionEquationIdMap.clear();
 
         // Clear constraint system
-        if (mpTMatrix != nullptr)
-            TSparseSpace::Clear(mpTMatrix);
-        if (mpConstantVector != nullptr)
-            TSparseSpace::Clear(mpConstantVector);
-        if (mpDeltaConstantVector != nullptr)
-            TSparseSpace::Clear(mpDeltaConstantVector);
+        mTMatrix.resize(0, 0, false);
```

Comment:
you will never be able to do this with trilinos matrices...

Reply:
Then the block Builder and solver will not be compatible too